### PR TITLE
chore: Handle Errors with ServiceError Helper

### PR DESCRIPTION
**Pull request by AlexcastroDev**
# What's new ?

- It helps to uniform the errors from API Services
- It check format and linting from all PR
- It stardandize the logger function
- It add Rate limit error
- it will not save render of rate limited calls
- It only cache success request

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -7,16 +7,19 @@ on:
...
+        uses: actions/checkout@v2
+      - name: Use Deno Version ${{ matrix.deno-version }}
         uses: denolib/setup-deno@v2
         with:
-          deno-version: "1.37.0"
-      - name: Run tests
-        run: deno test --allow-env
+          deno-version: ${{ matrix.deno-version }}
+      - name: Lint Deno Module
+        run: deno fmt --check
```
Comment (ryo-ma) [inline .github/workflows/testing.yml:23]: And, please run `deno lint`.

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -7,16 +7,19 @@ on:
 jobs:
   install-dependencies:
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        deno-version: [1.37.0]
```
Comment (ryo-ma) [inline .github/workflows/testing.yml:13]: The supported latest deno version by deno-vercel is v1.36.1.
https://github.com/vercel-community/deno/releases

Comment (AlexcastroDev) [reply .github/workflows/testing.yml:13]: The `vercel.json` already do it :)

Do i really need change the Runner ?

File: src/StaticRenderRegeneration/utils.ts [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
-    }
+  try {
+    return Deno.readFileSync(cacheFilePath);
+  } catch {
+    return null;
+  }
 }
 
 // https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest
 export async function generateUUID(message: string): Promise<string> {
```
Comment (ryo-ma) [inline src/StaticRenderRegeneration/utils.ts:21]: Is the function name a UUID?
Maybe digest or hash is correct?

Comment (AlexcastroDev) [reply src/StaticRenderRegeneration/utils.ts]: Done

File: .github/workflows/testing.yml (hunk 2 above)
Comment (AlexcastroDev) [reply .github/workflows/testing.yml]: changed

File: .github/workflows/testing.yml (hunk 1 above)
Comment (AlexcastroDev) [reply .github/workflows/testing.yml:23]: Done

File: .github/workflows/testing.yml (hunk 2 above)
Comment (ryo-ma) [reply .github/workflows/testing.yml]: OK

**Pull request closed without merging**